views/partials/employees-to-add.handlebars

```diff
@@ -0,0 +1,6 @@
+<section class="">
+<h3>
+    {{username}} <input type="checkbox" class="form-check-input" id="employeeAddCheck">
+    <label class="form-check-label" for="employeeAddCheck">Text for the checkbox</label>
```

Comment:
The {{username}} would go in the section that you have "text for the checkbox" so that it is the username showing up. And i believe you need to give it a value of {{id}} so that we can grab the value in the javascript file.